launchToolbar/js/launchToolbar_meta_prerelease.user.js

```diff
@@ -0,0 +1,26 @@
+// ==UserScript==
```

Comment:
this is perfect!  thank you for doing this.

Reply:
No problem :D